benchmarks/dynamo/common.py

```diff
@@ -2310,6 +2304,13 @@ def record_status(accuracy_status, dynamo_start_stats):
                         new_result = optimized_model_iter_fn(
                             *example_args, **example_kwargs
                         )
+                if self.args.export_aot_inductor:
```

Comment:
We should consolidate the logic with `if self.args.export:`, including inputs/outputs processing.